x-pack/plugins/security_solution/public/detections/containers/detection_engine/rules/api.ts

```diff
@@ -373,21 +374,57 @@ export const exportRules = async ({
 /**
  * Fetch rule execution events (e.g. status changes) from Event Log.
  *
- * @param ruleId string Saved Object ID of the rule (`rule.id`, not static `rule.rule_id`)
+ * @param ruleId Saved Object ID of the rule (`rule.id`, not static `rule.rule_id`)
+ * @param start Start daterange either in UTC ISO8601 or as datemath string (e.g. `2021-12-29T02:44:41.653Z` or `now-30`)
+ * @param end End daterange either in UTC ISO8601 or as datemath string (e.g. `2021-12-29T02:44:41.653Z` or `now/w`)
```

Comment:
Perhaps only accept date objects

Reply:
Keeping as `string` to support ease of use of providing datemath like `now-24h`. ISODateString validation is now happening on route decode as of 53cc6bf as well.